addon/mixins/click-outside.js

```diff
@@ -17,12 +17,12 @@ export default Mixin.create({
   setupOutsideClickListener: on('didInsertElement', function() {
     let clickHandler = this.get('handleOutsideClick').bind(this);
 
-    ['click', 'touchend'].forEach((e) => document.addEventListener(e, clickHandler));
+    ['click', 'touchstart'].forEach((e) => document.addEventListener(e, clickHandler));
```

Comment:
We can't do it this way. Afaik, `touchend` is triggered when the user releases the finger after touching the screen, which is how we detect a "click". This mixin (`click-outside.js`) has that purpose.

You can try attaching the tooltip close to a mobile scroll, like this:
https://stackoverflow.com/questions/2863547/javascript-scroll-event-for-iphone-ipad